metube/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: metube_metube_1
+      APP_PORT: 8081
+
+  metube:
+    image: ghcr.io/alexta69/metube:2023-11-16
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data:/downloads
+    environment:
+      PORT: 8081
```

Comment:
We can remove this environment directive completely since port 8081 is the default for the UI.